content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/configuration.md

```diff
@@ -16,18 +16,56 @@ Since both O3DE Gems and projects use the same CMake build functions, these inst
 
 ## Build setup
 
-The first step of enabling the Multiplayer Gem for a project is to make sure that the `<ProjectName>.Static` target of your project includes the correct dependencies. Make the following edits to the `<ProjectName>.Static` target:
+As usual, the first step of enabling the Multiplayer Gem is to make sure the gem is [added to the project](/docs/user-guide/project-config/add-remove-gems/).
+After adding the Multiplayer Gem to the project there are more steps that require editing some of the project's CMake and C++ files by hand. 
+### CMakeList.txt changes
+Make sure the `<ProjectName>.Static` target includes the correct dependencies. 
+Find the CMake file defining your project's static target; for example `<ProjectName>/Code/CMakeList.txt`. 
+Make the following edits to the `<ProjectName>.Static` target:
+1. In the `FILES_CMAKE` section, add `<projectname>_autogen_files.cmake`
+    ```cmake
+        ly_add_target(
+            NAME <ProjectName>.Static STATIC
+            ...
+            FILES_CMAKE
+                ...
+                <projectname>_autogen_files.cmake
+    ```
+    {{< note >}}Creating the new `<projectname>_autogen_files.cmake` file is covered below in the [Adding AutoGen CMake file](#adding_autogen_file) section.{{< /note >}}
 
 1. In the `BUILD_DEPENDENCIES PUBLIC` section, add `AZ::AzNetworking` and `Gem::Multiplayer`.
+   ```cmake
+    ly_add_target(
+        NAME <ProjectName>.Static STATIC
+        ...
+        BUILD_DEPENDENCIES
+            PUBLIC
+                AZ::AzNetworking
+                Gem::Multiplayer
+   ```
+
+    {{< note >}}`BUILD_DEPENDENCIES` might not contain a `PUBLIC` section, add it if necessary.{{< /note >}}
```

Comment:
Maybe, "If BUILD_DEPENDENCIES does not contain a public section, add it as above".

Reply:
I like that a lot, easier to read in an "If -> then" format. Fixed!